remedy.lic

```diff
@@ -82,6 +82,7 @@ class Remedy
     waitrt?
     case bput(command,
               'You need another splash of water',
+              'You need another splash of alcohol',
```

Comment:
Just water and alcohol, for now. I checked all the recipes.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

On 09/10/2016 09:54 PM, Robert Herbig wrote:
>
> In remedy.lic 
> <https://github.com/rpherbig/dr-scripts/pull/776#discussion_r78286696>:
>
> > @@ -82,6 +82,7 @@ class Remedy
> >      waitrt?
> >      case bput(command,
> >                'You need another splash of water',
> > +              'You need another splash of alcohol',
>
> Are there any more liquids than water and alcohol? If so, it might be 
> worth the time to future-proof this and match against |'You need 
> another splash of (.*)'|
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/rpherbig/dr-scripts/pull/776/files/ff4f0f743a88719df99147c8d373c887fe41441e#r78286696>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AU8XXBGMadu3yJ3IDqI2Bc8sfbIrA2vjks5qo19rgaJpZM4J531I>.
>